src/ucp/core/ucp_context.c

```diff
@@ -510,6 +510,18 @@ static ucs_config_field_t ucp_context_config_table[] = {
         {NULL}
   )},
 
+  {"GVA_ENABLE", "n",
+   "Enable Global VA infrastructure",
+   ucs_offsetof(ucp_context_config_t, gva_enable), UCS_CONFIG_TYPE_BOOL},
+
+  {"GVA_PIN", "y",
```

Comment:
maybe rename to GVA_MLOCK and describe specifically we're locking the memory with mlock()?